linalg/vector.cpp

```diff
@@ -371,15 +376,19 @@ void add(const double a, const Vector &x,
       const double *xp = x.data;
       const double *yp = y.data;
       double       *zp = z.data;
-      int            s = x.size;
+      const int      s = x.size;
 
 #ifdef MFEM_USE_OPENMP
       #pragma omp parallel for
-#endif
       for (int i = 0; i < s; i++)
       {
          zp[i] = a * xp[i] + b * yp[i];
       }
+#endif
```

Comment:
Same here.

Reply:
There are also a few more of them below that. These looked like bugs, but they might not actually be. Could you take a look and verify?